### PR TITLE
long polling client request

### PR DESCRIPTION
**Pull request by nagworld9**
## Description

Issue # 

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azurelinuxagent/ga/long_polling.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        self.longpolling_thread.start()
+
+    def daemon(self):
+        new_request = True
+        while new_request:
+            try:
+                logger.info("[HTTP Request(longPolling)] Timestamp:{0}".format(datetime.datetime.utcnow()))
+
+                timeout = 60
+                conn = httpclient.HTTPConnection("168.63.129.16", 32526, timeout=10)
```
Comment (kevinclark19a) [inline azurelinuxagent/ga/long_polling.py:58]: Why have two timeouts? why not just set this timeout to 60?

Comment (larohra) [inline azurelinuxagent/ga/long_polling.py:58]: NIT: `timeout` parameter needs to be updated here probably

Comment (nagworld9) [reply azurelinuxagent/ga/long_polling.py:58]: Timeout here just connection timeout is the number of seconds requests will wait for your client to establish a connection but doesn't represent how long client should wait for response. Unfortunately, this library doesn't support unlike requests library https://stackoverflow.com/questions/21965484/timeout-for-python-requests-get-entire-response?msclkid=75b3309fc72a11eca5ad6e51d51368ff

So, I had to come up with second timeout just for Read timeout. https://stackoverflow.com/questions/49704708/what-is-a-connection-timeout-during-a-http-request?msclkid=43fb8e06c72b11ecbc1c813c14fa0c6e

File: azurelinuxagent/ga/long_polling.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                url = "http://168.63.129.16:32526/longPolling"
+                conn.request(method="GET", url=url)
+                resp = conn.getresponse()
+
+                while timeout > 0 and resp is None:
```
[the remark this answers is not in the document]
Comment (nagworld9) [reply azurelinuxagent/ga/long_polling.py:63]: one for connection timeout specified in the call and second is read timeout how long client should wait to get a response back.

**Pull request merged**